openfisca_core/scripts/run_test.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import sys
+import os
+import importlib
+
+from openfisca_core.tools.test_runner import run_tests
+from openfisca_core.tools import detect_country_packages
+
+
+def build_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('path', help = "paths (files or directories) of tests to execute", nargs = '+')
+    parser.add_argument('-c', '--country_package', action = 'store', help = 'country package to use to run the test. If not provided, an automatic detection will be attempted by scanning the python packages installed in your environment which name contains the word "openfisca".')
+    parser.add_argument('-e', '--extensions', action = 'store', help = 'extensions to load, separated by commas (e.g -e "extension_1, extension_2")')
+    parser.add_argument('-r', '--reforms', action = 'store', help = 'reforms to apply to the country package, separated by commas (e.g -r openfisca_france.reforms.some_reform)')
+    parser.add_argument('-f', '--force', action = 'store_true', default = False,
+        help = 'force testing of tests with "ignore" flag and formulas belonging to "ignore_output_variables" list')
+    parser.add_argument('-n', '--name_filter', default = None, help = "partial name of tests to execute. Only tests with the given name_filter in their name, file name, or keywords will be run.")
+    parser.add_argument('-v', '--verbose', action = 'store_true', default = False, help = "increase output verbosity")
+    parser.add_argument('-m', '--relative_error_margin', help = u"relative error margin to use for tests that don't define any", action = 'store', type = float)
+    parser.add_argument('-M', '--absolute_error_margin', help = u"absolute error margin to use for tests that don't define any", action = 'store', type = float)
+
+    return parser
+
+
+def main():
+    parser = build_parser()
+    args = parser.parse_args()
+    logging.basicConfig(level = logging.DEBUG if args.verbose else logging.WARNING, stream = sys.stdout)
+
+    if args.country_package:
+        try:
+            country_package = importlib.import_module(args.country_package)
+        except:
+            print('ERROR: {} does not seem to be a valid Openfisca country package.'.format(args.country_package))
```

Comment:
It's usually better to keep quotes around an interpolated string, as it shows a potential empty string.